vc-verifier/kotlin/vcverifier/src/test/java/io/mosip/vercred/vcverifier/credentialverifier/validator/CwtValidatorTest.kt

```
@@ -0,0 +1,3 @@
+package io.mosip.vercred.vcverifier.credentialverifier.validator
+
+class CwtValidatorTest {}
```

detekt (1.23.8)

[warning] The class or object CwtValidatorTest is empty.

(detekt.empty-blocks.EmptyClassBlock)